tests/php/ORM/DBDateTest.php

```diff
@@ -48,42 +49,42 @@ protected function restoreNotices()
     public function testNiceDate()
     {
         $this->assertEquals(
-            '31/03/2008',
+            'Mar 31, 2008',
```

Comment:
The old date is effective the same as you'd get when using `IntlDateFormatter::SHORT`, though we're using `IntlDateFormatter::MEDIUM` here.

Steve and I had talked in person about making the default configurable.
That would be doable for `DateField_Disabled` in the above test, because the default for that is returned from `DateField::getDateLength()` as a fallback value. Though its notable that none of the tests for `DateField` itself failed... so I'm not sure about the value of making that change but can do if the reviewer wants me to.

That's not possible for `DBDate` in this test though, because the default value is a _default argument_ in the method signature of `DBDate::getFormatter()`.

Reply:
> Though its notable that none of the tests for DateField itself failed

Looks like that's because only the internal values were being validated in DateFieldTest,  rather than the formatted values like Nice()

>  so I'm not sure about the value of making that change but can do if the reviewer wants me to.

Reading https://github.com/php/php-src/issues/16127#issuecomment-2381832767 particularly around the "no contract" bit, I guess it's correct to not have config here and not attempt to retain backwards compatibility. Simply updating our unit tests is the correct response I think.